packages/boot/test/bootstrapTests/drivers.js

```diff
@@ -8,6 +8,10 @@ import { instanceNameFor } from '@agoric/inter-protocol/src/proposals/price-feed
 
 import { boardSlottingMarshaller } from '@agoric/vats/tools/board-utils.js';
 
+/**
+ * @typedef {Awaited<ReturnType<Awaited<ReturnType<typeof makeWalletFactoryDriver>>['provideSmartWallet']>>} SmartWalletDriver
+ */
```

Comment:
consider,
```suggestion
/**
 * @typedef {Awaited<ReturnType<typeof makeWalletFactoryDriver>} WalletFactoryDriver
 * @typedef {Awaited<ReturnType<WalletFactoryDriver['provideSmartWallet']>>} SmartWalletDriver
 */
```
also https://github.com/Agoric/agoric-sdk/pull/8313/files#diff-115c511aa138ad34dd58b09e625da444a782540610c55e64bb3315d7a9291134

Reply:
That's fine.  This was really about just making `tsc` shut up.